<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>names</class>
 <widget class="QDialog" name="names">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Names</string>
  </property>
  <widget class="QSpinBox" name="userId1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="name1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="userIdLabel1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>User ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>names</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>names</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
